packages/manager/apps/dedicated/client/app/app.js

```diff
@@ -240,4 +241,10 @@ angular
         });
       });
   })
-  .constant('UNIVERSE', 'DEDICATED');
+  .constant('UNIVERSE', 'DEDICATED')
+  .config(/* @ngInject */($httpProvider) => {
+    $httpProvider.interceptors.push('OvhNgRequestTaggerInterceptor');
+  })
+  .run(/* @ngInject */ (OvhNgRequestTaggerInterceptor) => {
+    OvhNgRequestTaggerInterceptor.setHeaderVersion(Environment.getVersion());
```

Comment:
Can this be handled by `CoreConfig` like the region ?

Reply:
the header version should be configured in `@ovh-ux/manager-core`